Tests/test_SeqIO_features.py

```diff
@@ -501,21 +500,19 @@ def test_qualifiers(self):
         """
         f = SeqFeature(FeatureLocation(10, 20), strand=+1, type="CDS")
         self.assertEqual(f.qualifiers, {})
-        f = SeqFeature(FeatureLocation(10, 20), strand=+1, type="CDS",
-                qualifiers={"test": ["a test"]})
+        f = SeqFeature(FeatureLocation(10, 20), strand=+1, type="CDS", qualifiers={"test": ["a test"]})
         self.assertEqual(f.qualifiers["test"], ["a test"])
 
 
 class FeatureWriting(unittest.TestCase):
     def setUp(self):
-        self.record = SeqRecord(Seq("ACGT" * 100, generic_dna),
-                                id="Test", name="Test", description="Test")
+        self.record = SeqRecord(Seq("ACGT" * 100, generic_dna), id="Test", name="Test", description="Test")
 
-    def write_read_check(self, format):
+    def write_read_check(self, in_format):
```

Comment:
I don't understand this variable change name - the format argument is used to write a file and then read it back - it isn't an input file format.